libr/util/swap.c

```diff
@@ -1,33 +1,33 @@
 #include <r_util.h>
 
 
-R_API ut16 r_swap_ut16(ut16 val) {
+R_API inline ut16 r_swap_ut16(ut16 val) {
     return (val << 8) | (val >> 8 );
 }
 
-R_API st16 r_swap_st16(st16 val) {
+R_API inline st16 r_swap_st16(st16 val) {
     val = ((val << 8) & 0xFF00FF00 ) | ((val >> 8) & 0xFF00FF );
     return (val << 16) | (val >> 16);
 }
 
-R_API ut32 r_swap_ut32(ut32 val) {
+R_API inline ut32 r_swap_ut32(ut32 val) {
     val = ((val << 8) & 0xFF00FF00 ) | ((val >> 8) & 0xFF00FF );
     return (val << 16) | (val >> 16);
 }
 
-R_API st32 r_swap_st32(st32 val) {
+R_API inline st32 r_swap_st32(st32 val) {
     val = ((val << 8) & 0xFF00FF00) | ((val >> 8) & 0xFF00FF );
     return (val << 16) | ((val >> 16) & 0xFFFF);
 }
 
 
-R_API ut64 r_swap_ut64(ut64 val) {
+R_API inline ut64 r_swap_ut64(ut64 val) {
     val = ((val << 8) & 0xFF00FF00FF00FF00ULL ) | ((val >> 8) & 0x00FF00FF00FF00FFULL );
     val = ((val << 16) & 0xFFFF0000FFFF0000ULL ) | ((val >> 16) & 0x0000FFFF0000FFFFULL );
     return (val << 32) | (val >> 32);
 }
 
-R_API st64 r_swap_st64(st64 val) {
+R_API inline st64 r_swap_st64(st64 val) {
```

Comment:
R_API = public
inline = not public

this is not correct. inlined functions should be defined static and implemented in the .h